### PR TITLE
Pimp my drip

### PR DESCRIPTION
**Pull request by McRamon**
## What Does This PR Do
Adds a very expensive coat for crewmembers to wear (costs 1000 cerdits in cargo)

## Why It's Good For The Game
Its very drippy

## Images of changes
image

## Testing
testing

## Changelog
:cl:
add: New drip for rich guys - expensive coat, can only be ordered i ncargo for 1000 credits
/:cl:

### CODE REVIEW

**Comment by ppi13** [comment]
Is it so that skrell, taj, vulp, slime, plasma and nian cant wear this? If so, I fully support it!

**Comment by McRamon** [comment]
> Is it so that skrell, taj, vulp, slime, plasma and nian cant wear this? If so, I fully support it!

they use human sprites :(

File: code/modules/clothing/suits/misc_suits.dm [hunk 1]
```diff
@@ -386,6 +386,17 @@
...
+/obj/item/clothing/suit/pimpcoat
+	name = "expensive coat"
+	desc = "Very fluffy pink coat, made out of very expensive fur (clearly)."
+	icon_state = "pimpcoat"
+	item_state = "pimpcoat"
+	blood_overlay_type = "armor"
+	body_parts_covered = UPPER_TORSO|ARMS|LOWER_TORSO
+	allowed = list(/obj/item/tank/internals/emergency_oxygen)
+	cold_protection = UPPER_TORSO | LOWER_TORSO | ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
```
Comment (ppi13) [inline code/modules/clothing/suits/misc_suits.dm:398]:
```suggestion
	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
	species_restricted = list("Human", "Vox", "Drask", "Grey", "Kidan" )
```

**Review by Sirryan2002 — APPROVED** [review]

**Review by Sirryan2002 — APPROVED** [review]

File: code/modules/clothing/suits/misc_suits.dm [hunk 2]
```diff
@@ -386,6 +386,17 @@
 
 	sprite_sheets = list("Vox" = 'icons/mob/clothing/species/vox/head.dmi')
 
+/obj/item/clothing/suit/pimpcoat
+	name = "expensive coat"
+	desc = "Very fluffy pink coat, made out of very expensive fur (clearly)."
+	icon_state = "pimpcoat"
+	item_state = "pimpcoat"
+	blood_overlay_type = "armor"
+	body_parts_covered = UPPER_TORSO|ARMS|LOWER_TORSO
```
Comment (S34NW) [inline code/modules/clothing/suits/misc_suits.dm:395]:
```suggestion
	body_parts_covered = UPPER_TORSO | LOWER_TORSO | ARMS
```

**Review by S34NW — CHANGES_REQUESTED** [review]
consistency

**Review by S34NW — APPROVED** [review]

**Review by S34NW — CHANGES_REQUESTED** [review]
Three weeks with no de-conflict, please get this done so we can merge it

**Review by S34NW — APPROVED** [review]

**Pull request merged**